frontend/src/components/associations/Home.tsx

```diff
@@ -2,16 +2,42 @@ import React from "react";
 import { PageTitle } from "../utils/PageTitle";
 import Container from "react-bootstrap/Container";
 import { Loading } from "../utils/Loading";
+import { api, useBetterPaginatedQuery } from "../../services/apiService";
+import { ErrorMessage } from "../utils/ErrorPage";
+import { AssociationPageRenderer } from "./page/Show";
 
 export const AssociationHome = ({ association }) => {
```

Comment:
I have a TypeError: association.myRole.permissions is undefined when going to this page logged in as `19simple`.